STTBot/events/app_mention.py

```diff
@@ -394,6 +402,16 @@ def _ret_success_blocks(blocks, suc_message, say):
         "help": "Adds reactions for pin showdown",
         "func": _cmd_poll_react
     },
+    {
+        "cmd": "STT",
+        "sub_cmd": "Draft",
+        "args": [
+            "list"
```

Comment:
`list_of_users` might be a better variable name x

Reply:
agree